README.md

```diff
@@ -56,6 +56,19 @@ npm install cron
 
 With the introduction of TypeScript in version 3 and alignment with UNIX cron patterns, a few changes have been made:
 
+<details>
+  <summary>Migrating from v3 to v4</summary>
+
+### Dropped Node version
+
+Node v16 is no longer supported. Upgrade your Node installation to Node v18 (LTS) or above
```

Comment:
I would remove the "LTS" bit since the current LTS is version 22, that might be confusing.